Don't have a Windows-machine to build on atm. but hopefully I caught most of them.

@iSazonov Any tips regarding https://github.com/PowerShell/PowerShell/pull/17795/commits/31408ebc206304b2bf4325cac1cf21025490ed8f ? 
When building in devcontainer with `-Runtime win7-x64` and changing them to `securityDescHandle?.Value.Free();` it failed with:

```
/workspaces/PowerShell/src/System.Management.Automation/engine/remoting/common/RunspaceConnectionInfo.cs(2850,32): error CS1061: 'GCHandle' does not contain a definition for 'Value' and no accessible extension method 'Value' accepting a first argument of type 'GCHandle' could be found (are you missing a using directive or an assembly reference?) [/workspaces/PowerShell/src/System.Management.Automation/System.Management.Automation.csproj]
/workspaces/PowerShell/src/System.Management.Automation/engine/remoting/common/RemoteSessionNamedPipe.cs(507,32): error CS1061: 'GCHandle' does not contain a definition for 'Value' and no accessible extension method 'Value' accepting a first argument of type 'GCHandle' could be found (are you missing a using directive or an assembly reference?) [/workspaces/PowerShell/src/System.Management.Automation/System.Management.Automation.csproj]
```

--- a/src/Microsoft.WSMan.Management/WsManHelper.cs
+++ b/src/Microsoft.WSMan.Management/WsManHelper.cs
@@ -370,10 +370,7 @@ namespace Microsoft.WSMan.Management
             }
             finally
             {
-                // _sr?.Close();
                 _sr?.Dispose();
-
-                // _fs?.Close();
                 _fs?.Dispose();
             }
 

--- a/src/System.Management.Automation/engine/remoting/common/RemoteSessionNamedPipe.cs
+++ b/src/System.Management.Automation/engine/remoting/common/RemoteSessionNamedPipe.cs
@@ -504,12 +504,7 @@ namespace System.Management.Automation.Remoting
                 securityAttributes);
 
             int lastError = Marshal.GetLastWin32Error();
-#pragma warning disable IDE0031
-            if (securityDescHandle != null)
-            {
-                securityDescHandle.Value.Free();
-            }
-#pragma warning restore IDE0031
+            securityDescHandle?.Free();
 
             if (pipeHandle.IsInvalid)
             {

--- a/src/System.Management.Automation/engine/remoting/common/RunspaceConnectionInfo.cs
+++ b/src/System.Management.Automation/engine/remoting/common/RunspaceConnectionInfo.cs
@@ -2847,12 +2847,7 @@ namespace System.Management.Automation.Runspaces
                 securityAttributes);
 
             int lastError = Marshal.GetLastWin32Error();
-#pragma warning disable IDE0031
-            if (securityDescHandle != null)
-            {
-                securityDescHandle.Value.Free();
-            }
-#pragma warning restore IDE0031
+            securityDescHandle?.Free();
 
             if (pipeHandle.IsInvalid)
             {

